rb/spec/integration/selenium/webdriver/chromium/driver_spec.rb

```diff
@@ -22,7 +22,7 @@
 module Selenium
   module WebDriver
     module Chrome
```

Comment:
This spec now lives in `chromium` directory but the module is still `Chrome`. Can we solve this mismatch somehow?

Reply:
I just went with the superclass; seemed better than creating 2 subclasses and a superclass in the library.